libs/qCC_db/src/ccPointCloud.cpp

```diff
@@ -5803,36 +5803,40 @@ bool ccPointCloud::drawNormals(CC_DRAW_CONTEXT& context)
 
 	if (m_programDrawNormals.isNull())
 	{
-		m_programDrawNormals = QSharedPointer<QOpenGLShaderProgram>(new QOpenGLShaderProgram(context.qGLContext));
+		m_programDrawNormals.reset(new QOpenGLShaderProgram(context.qGLContext));
```

Comment:
So each point cloud has its own program? Can't you share one for all the clouds?

Reply:
Not sure about my changes here as the IDE is now complaining about "non-POD static". What do you think about it?